content/build/cache/_index.md

```diff
@@ -50,7 +50,11 @@ anything differently, they still need to re-run.
 
 > **Note**
 >
-> Suppose you have a `RUN apt-get update && apt-get upgrade -y` step in your
+> Suppose you have a
+> ```text
+> RUN apt-get update && apt-get upgrade -y
+> ``` 
+> step in your
```

Comment:
This breaks it into two paragraphs, or three block elements, which isn't appropriate for how it's currently written. If we change the formatting here it needs a rewrite.

Reply:
Thanks for reviewing ! So what is the possible solution?, guide me I will rewrite it.